charts/alfresco-connector-ms365/templates/_helpers-env.tpl

```diff
@@ -0,0 +1,24 @@
+{{/*
+Set environment variables necessary for configmap
+*/}}
+{{- define "alfresco-connector-ms365.repo-ms365.env" -}}
```

Comment:
remove `repo-ms365` from the name or move L17-24 back to deployment template because those are not really related to repository configuration